src/modules/invite/invite.controller.ts

```diff
@@ -41,4 +44,29 @@ export class InviteController {
       }
     }
   }
+
+  @Post(':orgId/invite/:inviteId/refresh')
+  async refreshInvitation(@Param('orgId') orgId: string, @Param('inviteId') inviteId: string) {
+    if (isUUID(orgId, 4)) {
+      throw new CustomHttpException(
+        {
+          status_code: 400,
+          error: 'Bad Request',
+          message: 'Invalid organisation ID format',
```

Comment:
Use constants where possible to make code cleaner and maintainable. Hardcoded strings just leaves room for errors